lib/kitchen/directions/bake_appendix.rb

```diff
@@ -16,8 +16,9 @@ def self.v1(page:, number:)
         # Make a section with data-depth of X have a header level of X+1
         page.search('section').each do |section|
           title = section.first!("[data-type='title']")
-          title.name = "h#{section['data-depth'].to_i + 1}"
+          title.name = "h#{section['data-depth'].to_i + 1}"  # doesn't work
```

Comment:
@jpslav Good catch asking for me to test the section header -- tests are now failing because `section['data-depth']` isn't retrieving the right value. I took a stab at fixing this myself, but couldn't hack it.

My investigation: 
- `section['data-depth']` is empty, and `section['data-depth'].to_i` returns `0` no matter what the depth is. 
- The syntax of this implies that `section` is a hash with key `data-depth`, but `section.class` is `ElementEnumerator`
- The Ruby hash methods (`key`, `include`?) don't exist on `section`. 

Any advice?

Reply:
The `section` tag in your test needs to have a `data-depth` attribute, e.g. `data-depth=1`.  Here's what one looks like in real assembled book XHTML:

```xml
<section data-depth="1" id="auto_m68662_eip-260">
```

`section['blah']` is legit, the `[]` method on the `Kitchen::Element` class looks up the value for an attribute on the element.  Since the attribute is missing, `section['data-depth']` is `nil`, and `nil.to_i` is `0` for better or worse.

Reply:
does kind of beg the question of what to do if the `data-depth` tag isn't there...